turbinia/notify.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from turbinia import config
+
+#Sends notfications via email
+def sendmail(message):
+
+    log=logging.getLogger('turbinia')
+    try:
+        EMAIL_NOTIFICATIONS = config.EMAIL_NOTIFCATIONS
+        if EMAIL_NOTIFICATIONS == 'true':
+            EMAIL_HOST_ADDRESS = config.EMAIL_HOST_ADDRESS
+            EMAIL_PORT = int(config.EMAIL_PORT)
+
+            EMAIL_ADDRESS = config.EMAIL_ADDRESS
+            EMAIL_PASSWORD = config.EMAIL_PASSWORD
+
+            RECIEVING_ADDRESS = config.RECIEVING_ADDRESS
+            #Imports SMTP and  MIME(The format used for emails)
+            from email.mime.multipart import MIMEMultipart
+            from email.mime.text import MIMEText
+            import smtplib
+
+            #Puts message in MIME format
+            msg = MIMEMultipart()
+            msg['From'] = EMAIL_ADDRESS
+            msg['To'] = RECIEVING_ADDRESS
+            msg['Subject'] = 'Notifcation from Turbina'
+            msg.attach(MIMEText(message))
+
+            #start SMTP
+            server = smtplib.SMTP(EMAIL_HOST_ADDRESS,EMAIL_PORT)
+
+            # identify to mail server
+            server.ehlo()
+
+            # secure email using TLS
+            server.starttls()
+
+            # reidentify and login to mail account
+            server.ehlo()
+            server.login(EMAIL_ADDRESS, EMAIL_PASSWORD)
+
+            server.sendmail(EMAIL_ADDRESS, RECIEVING_ADDRESS,msg.as_string())
+
+            #terminate connection
+            server.quit()
+            log.info('Email notification sent')
+        else:
+            log.info('Email notifications are disabled')
+            pass
+    except Exception as e:
```

Comment:
This is a pretty broad exception, can we change it to smtplib.SMTPException, which looks like it's the base class for all of the smtp exceptions?

Reply:
I could, But then in cases where the user made an error when making the config(say that the user tried doing something weird like putting an interger as the EMAIL_ADDRESS , or something) it could cause more problems than it needs too.

Reply:
Generally catching a very broad exception in python is generally discouraged, and we try to only do that when absolutely necessary.   See the "catch-all" section in the style guide for more details:
http://google.github.io/styleguide/pyguide.html#244-decision

I would suggest either catching the exceptions you think might be thrown and/or adding checks or try/except blocks around the code segments that are risky.

Reply:
Yeh your right, I'll edit it.